test/client/getLedgerIndex.ts

```diff
@@ -0,0 +1,15 @@
+import { assert } from 'chai'
+
+import rippled from '../fixtures/rippled'
+import { setupClient, teardownClient } from '../setupClient'
+
+describe('client.getLedgerIndex', function () {
+  beforeEach(setupClient)
+  afterEach(teardownClient)
+
+  it('getLedgerIndex', async function () {
+    this.mockRippled.addResponse('ledger', rippled.ledger.normal)
+    const fee = await this.client.getLedgerIndex()
```

Comment:
Uh, `fee` is probably a misleading variable name here, right?

Reply:
whoops, copy-paste error